e2eAIOK/ModelAdapter/example/Distiller_ResNet18_from_VIT_on_CIFAR100_save_logits.ipynb

```diff
@@ -0,0 +1,244 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6cb2bd5d",
+   "metadata": {},
+   "source": [
+    "# Save logits from VIT with Distiller on CIFAR100\n",
+    "Distiller can transfer knowledge from a heavy model (teacher) to a light one (student) with different structure.\n",
+    "\n",
+    "* Teacher is a large model pretrained on specific dataset, which contains sufficient knowledge for this task, while the student model has much smaller structure. Distiller trains the student not only on the dataset, but also with the help of teacher’s knowledge.\n",
+    "* Distiller can take use of the knowledge from the existing pretrained large models but use much less training time. It can also significantly improve the converge  speed and predicting accuracy of a small model, which is very helpful for inference.\n",
+    "![Distiller](../doc/imgs/distiller.png)\n",
+    "\n",
+    "However, during the distillation process, teacher forwarding usually takes a lot of time. We can use logits saving function in distiller to save predictions from teacher in adavance, then lots of time can be saved during student training.\n",
+    "\n",
+    "## Notebook Content\n",
+    "In this notebook, we will show how to use logits saving function, and here we still take VIT as an example.\n",
+    "\n",
+    "To enable saving logits function, we just need to add two steps in original pipeline:\n",
+    "- Wrap train_dataset with DataWrapper\n",
+    "- Call prepare_logits() in Distiller and exit\n",
+    "\n",
+    "Note: We need to save all the data without any sampling, make sure you have disable \"channel_last\" or \"sampler\" in dataloader, which can avoid data lossing in later logits using process."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e51d390c",
+   "metadata": {},
+   "source": [
+    "## Environment Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "f878badd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import torch\n",
+    "from torchvision import transforms,datasets\n",
+    "from torch.utils.data import DataLoader\n",
+    "import torch.optim as optim\n",
+    "import timm\n",
+    "import transformers\n",
+    "import datetime"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "7bc00450",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.append(\"/home/vmagent/app/TLK/frameworks.bigdata.AIDK/AIDK/\")\n",
```

Comment:
Since project name is e2eaiok, `frameworks.bigdata.AIDK/AIDK` need to be changed.

Reply:
Updated